docs/Using-Fleet/Application-security.md

```diff
@@ -13,16 +13,17 @@ The Fleet community follows best practices when coding. Here are some of the way
 
 ### Describe your secure coding practices, including code reviews, use of static/dynamic security testing tools, 3rd party scans/reviews.
 
-Code commits to Fleet go through a series of tests, including SAST (static application security testing). 
+Code commits to Fleet go through a series of tests, including SAST (static application security
+testing). We use a combination of tools including [gosec](https://github.com/securego/gosec) and
+[CodeQL](https://codeql.github.com/) for this purpose.
 
-Every piece of code that is merged into Fleet is reviewed by at least one other engineer before merging. 
+Every piece of code that is merged into Fleet is reviewed by at least one other engineer before
+merging.  This is enforced via branch protection on the main branch.
```

Comment:
```suggestion
Merging,  we enforce this via branch protection on the main branch.
```

Reply:
Not sure why Merging with an uppercase `M`? It's at the beginning of a line but not a sentence, possibly a wrapping issue @Desmi-Dizney?

Reply:
@GuillaumeRoss, if that's the case I'm confused because it already says "merged" in this sentence. Perhaps a rewrite for clarity would be helpful.

Reply:
@Desmi-Dizney rewrote! Can I haz re-review? :)